app/controllers/crops_controller.rb

```diff
@@ -215,6 +215,15 @@ def destroy
   private
 
   def crop_params
-    params.require(:crop).permit(:en_wikipedia_url, :name, :parent_id, :creator_id, :approval_status, :request_notes, :reason_for_rejection, :rejection_notes, scientific_names_attributes: [:scientific_name, :_destroy, :id])
+    params.require(:crop).permit(:en_wikipedia_url,
```

Comment:
I don't have a strong opinion either way, but it seems like the way @Br3nda did it here is more likely to result in nested things hitting the line length limit anyway. I've certainly hit the 80 char limit in Python doing it this way before. (And I do have the impression the say it's done here is what's common in those languages, so I can only guess the people who wrote PEP-8 and whatever Ruby's version is are shaking their fists and saying "don't nest so deep!" at me)